config/files/server/usr/etc/ssh/sshd_config.d/99-hardening.conf

```diff
@@ -20,4 +26,32 @@ TCPKeepAlive no
 ClientAliveCountMax 2
 
 # Forbid sshing as root
-PermitRootLogin no
\ No newline at end of file
+PermitRootLogin no
+
+# Enable Pubkey Authentification
```

Comment:
is this not default?

Reply:
I could remove it to keep aligned with "as upstream as possible"